docs/arvo/network.md

```diff
@@ -88,7 +86,7 @@ line).
 The second thing `++poke-noun` produces is our state.  `+>.$`
 refers to a particular address in our subject where our formal
 app state is stored.  It'll become clear why this is later on,
-but for now pretend that `+>.$` is a magic invocation that means
+but for now pretend that `+>.$` is a magic incantation that means
 "app state".
```

Comment:
"incantation" actually feels a little better to me.  "invocation" implies "calling" in my mind, and `+>.$` isn't really doing that.  Not much of a difference, though.